CPFCNPJTools/Tests/CPFManagerTests.swift

```diff
@@ -8,71 +8,71 @@ class CPFValidatorTests: XCTestCase {
     }
     
     func testValidateWhenTheCPFIsNilOrEmpty() {
-        let sut = CPFValidator()
+        let sut = CPFManager()
```

Comment:
extract this `sut` into a variable outside the function to use in all test case

Reply:
Done!